truststore_nss.go

```diff
@@ -68,8 +69,23 @@ func (m *mkcert) checkNSS() bool {
 func (m *mkcert) installNSS() bool {
 	if m.forEachNSSProfile(func(profile string) {
 		cmd := exec.Command(certutilPath, "-A", "-d", profile, "-t", "C,,", "-n", m.caUniqueName(), "-i", filepath.Join(m.CAROOT, rootName))
-		out, err := cmd.CombinedOutput()
-		fatalIfCmdErr(err, "certutil -A", out)
+
+		// Wait for cmd incase a password prompt is given
+		errChan := make(chan error)
+		timeout := time.After(1*time.Second)
+		go func() {
+			out, err := cmd.CombinedOutput()
+			fatalIfCmdErr(err, "certutil -A", out)
+			errChan <- err
+		}()
+		for { // block until errChan gets a result
+			select {
+			case <-errChan:
+				return // next profile
+			case <-timeout:
+				log.Print("\n(hint, this is your Firefox master password))")
```

Comment:
Firefox is now calling it a primary password rather than a master password: <https://support.mozilla.org/en-US/kb/primary-password-replacing-master-password>.

Reply:
Updated, thanks.